src/styles/index.scss

```diff
@@ -1,3 +1,119 @@
+@import './utils/variables';
+@import './stars';
+
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
The `body` tag already has this property.
Also, add only the `box-sizing` property and nothing else to the `*` tag(Applies to all future projects)